src/workerd/server/workerd.c++

```diff
@@ -20,6 +20,10 @@
 #include <openssl/rand.h>
 #include <workerd/io/compatibility-date.capnp.h>
 
+#ifdef WORKERD_ENABLE_WEBGPU
```

Comment:
Non-blocking nit... given that this is still experimental/poc, let's make the gate here `WORKERD_EXPERIMENTAL_ENABLE_WEBGPU`... I just want to make it super clear just by looking at it what the status of this is.